.changeset/accounttoken_newaccounttoken.md

```diff
@@ -0,0 +1,9 @@
+---
+default: major
+---
+
+# `Account.Token` -> `NewAccountToken`
+
+#340 by @chris124567
+
+This method does not use any of the fields on `Account`
```

Comment:
Auto generated changeset. Better to just do "Change `Account.Token` method to `NewAccountToken` function" or similar.